### PR TITLE
update the README

### PR DESCRIPTION
**Pull request by MaximilianMeister**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,13 +21,29 @@ was originally developed by the [Dell CloudEdge Solutions Team](http://dell.com/
 ## Install
 
 To install this simple rubygem you can check your package manager if it is
-available for you, alternativly you can just install the ruby gem if you have a
+available for you, alternatively you can just install the rubygem if you have a
```
Comment (vuntz) [inline README.md]: I think ruby gem in two words is correct for a sentence :-)

Comment (MaximilianMeister) [inline README.md]: one line above it's "rubygem" again :-) but I will change all of them to "ruby gem" if preferred

File: README.md [hunk 2]
```diff
@@ -20,14 +20,30 @@ was originally developed by the [Dell CloudEdge Solutions Team](http://dell.com/
...
 
 ```
 gem install crowbar-client
 ```
 
+## Configuration
+
+`crowbar-client` can be configured by placing a `.crowbarrc` file in your `$HOME` directory.
+The default configuration looks like:
+```
```
Comment (tboerger) [inline README.md]: Please add an empty line

File: README.md [hunk 3]
```diff
@@ -20,14 +20,30 @@ was originally developed by the [Dell CloudEdge Solutions Team](http://dell.com/
...
+[default]
+server = http://localhost
+username = crowbar
+password = crowbar
+anonymous = false
+```
+
+## Usage
+
+`crowbarctl help`
```
Comment (tboerger) [inline README.md]: Please use also the expanded variant with ```

File: README.md (hunk 2 above)
Comment (tboerger) [inline README.md]: Or even better just link to https://github.com/crowbar/crowbar-client/blob/master/config/crowbarrc

File: README.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
 # Crowbar: Client
 
-This is an experimental implementation to replace the current CLI scripts of
-the crowbar barclamps. This tool is intended to be installable standalone,
-that way everybody who got access to the Crowbar admin network should be able
-to manage Crowbar from remote.
+A complete reimplementation of the old CLI scripts of the Crowbar Barclamps.
```
Comment (dirkmueller) [inline README.md]: Isn't there a "This is.." missing?

Comment (tboerger) [inline README.md]: Yes

Comment (tboerger) [inline README.md]: > This is a complete reimplementation of the old CLI scripts of Crowbar. It is intended...

Comment (MaximilianMeister) [inline README.md]: >  Isn't there a "This is.." missing?

I thought the "This is" is obvious, so I removed it. But if you prefer that, I will put it back

> This tool is a complete reimplementation of the old CLI scripts of Crowbar. It is intended...

even though the next sentence starts with "This tool..." as well?

Comment (tboerger) [inline README.md]: > even though the next sentence starts with "This tool..." as well?

Not anymore if you take my sentence :D

Comment (MaximilianMeister) [inline README.md]: ouh sorry, reading too fast :-)

**Comment by tboerger** [comment]
:+1:

**Comment by rsalevsky** [comment]
:+1:

**Pull request merged**